bot.py

```diff
@@ -259,7 +264,27 @@ def is_correct_response(message: Message):
 #############
 @bot.listen()
 async def on_ready():
-    log.info(f"Bot is ready: logged in as {bot.user.name} ({bot.user.id})")
+    # badly implement status lmao
+    #str(DISCORD_TYPE.lower())
+
+    if(DISCORD_TYPE == "watching"):
+        await bot.change_presence(status=discord.Status(DISCORD_STATUS), activity=discord.Activity(type=discord.ActivityType.watching, name=DISCORD_STATUS_NAME))
+
+    elif(DISCORD_TYPE == "playing"):
+        await bot.change_presence(status=discord.Status(DISCORD_STATUS), activity=discord.Game(name=DISCORD_STATUS_NAME))
+
+    elif(DISCORD_TYPE == "listening"):
+        await bot.change_presence(status=discord.Status(DISCORD_STATUS), activity=discord.Activity(type=discord.ActivityType.listening, name=DISCORD_STATUS_NAME))
+
+    elif(DISCORD_TYPE == "streaming"):
+        await bot.change_presence(status=discord.Status(DISCORD_STATUS), activity=discord.Streaming(name=DISCORD_STATUS_NAME, url=DISCORD_TWITCH))
+
+    log.info("Bot is ready: logged in as {bot.user.name} ({bot.user.id})")
+    log.info("Checking for Pterodactyl...")
+    if(str(sys.argv[1]) == 'ptero'):
+        log.info("Pterodactyl Detected!")
+    else:
+        log.info("Pterodactyl not found.")
```

Comment:
- `client` is redundant here (along with its import)
- a bit nitpicky, but would you mind removing the general `import discord` in favor of adding more specific imports above on line 10 (and renaming `discord.Status`, `discord.Streaming` to just `Status` and `Streaming`)? Just for consistency :)